app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Http\Request;
+use App\Http\Resources\User as UserResource;
+use App\Http\Resources\Users as UsersResource;
+
+class UserController extends Controller
```

Comment:
Good, just need to keep in mind down the road that we need to add a API key to the .env file so that no one can make calls to our API without having proper credentials